user-interface/0001-WALL-wallet.md

```diff
@@ -58,18 +58,26 @@ When a dapp sends a transaction to the wallet for signing and broadcast, I...
 
 ... so I can verify that the transaction being sent is the one I want
 
-## Transactions
+## Transaction log
 When thinking about a recent or specific transaction, I ...
 
-- I can see a history of transactions the wallet has signed. As read from the local app (Current "session" only, as persistent data storage has other requirements (see commented out ACs)) (<a name="0001-WALL-034" href="#0001-WALL-034">0001-WALL-034</a>)
-- I can see pending transactions (Transactions I have not yet confirmed/rejected) (<a name="0001-WALL-035" href="#0001-WALL-035">0001-WALL-035</a>)
-- I can see transactions that were rejected by the wallet user (me) (<a name="0001-WALL-037" href="#0001-WALL-037">0001-WALL-037</a>)
+- I can navigate between the transaction list and wallet list (tabs) (<a name="0001-WALL-033" href="#0001-WALL-033">0001-WALL-033</a>)
```

Comment:
comment (non-blocking): This sounds like an implementation detail

Reply:
that's not a problem in its self but so far these AC have attempted to be solution agnostic

Reply:
True, removed.